File: modules/cubex-i18n/src/main/java/org/cubexmc/i18n/I18nServices.java
package org.cubexmc.i18n;

import org.cubexmc.core.CubexPlugin;

public final class I18nServices {

    private I18nServices() {
    }

    public static I18nService create(CubexPlugin plugin, I18nOptions options) {
        return new SimpleI18nService(plugin, options == null ? I18nOptions.create() : options);
    }
}

File: modules/cubex-i18n/src/main/kotlin/org/cubexmc/i18n/I18nServices.kt
package org.cubexmc.i18n

import org.cubexmc.core.CubexPlugin

object I18nServices {
    @JvmStatic
    fun create(plugin: CubexPlugin, options: I18nOptions?): I18nService =
        SimpleI18nService(plugin, options ?: I18nOptions.create())
}
